.changeset/fuzzy-emus-rush.md

```diff
@@ -0,0 +1,12 @@
+---
+"@kaizen/components": major
```

Comment:
I think this is a `minor` we're introducing new Icon components and the original `<Icon />` component isn't being updated. (or rather non existent here)